test/frontend.js

```diff
@@ -144,7 +144,7 @@ describe('Frontend', function () {
 
   describe('Sign up Test', function () {
     it('Should register User', function () {
-      this.timeout(10000)
+      this.timeout(50000)
```

Comment:
do we really need this timeout or it was just to debug?

Reply:
I think this was just to debug Travis, I can revert it back to 10000. 👍